vnext/proposals/active/mouse-reconcile-desktop.md

```diff
@@ -0,0 +1,107 @@
+- Ensuring mouse behavior is consistent across the two platforms we support
+- Once our two desktop platforms are aligned with React Native Web, we upstream this implementation to core [issue TBD]()
+
+## `hovered`
+Property on `Pressable` that determines whether or not the mouse is currently within the bounds of the designated content.
+
+| TYPE | REQUIRED | PLATFORM |
+|:--:|:--|:--|
+| boolean | Yes | Windows, macOS |
+
+[React Native Web](https://github.com/necolas/react-native-web/commit/1afd9de0455965a852bad42ad2ffab957f7ffca0#diff-9fe3ab194c72e9837a8ffc3e7c44f33d) introduced  `hovered` on `Pressable` to enable common mouse scenarios. In order to align and make this complete the work is as follows:
+
+- Windows has added `hovered` and marked `onMouseEnter/Leave` as to be deprecated [issue TBD]()
+- Add `hovered` to `Pressable` [issue TBD]()
+- macOS has added 'hovered' and marked their View component implementation of `onMouseEnter/Leave` as deprecated [issue TBD]()
+
+### `hovered` example
+
+``` js
+<Pressable style={({ hovered, pressed, focused }) => {
+          console.log(focused);
+          let backgroundColor = 'white';
+          if (hovered) {
+            backgroundColor = 'lightgray';
+          }
+          if (pressed) {
+            backgroundColor = 'lightblue';
+          }
+          return {
+            padding: 10,
+            margin: 10,
+            borderWidth: 1,
+            borderColor: focused ? 'red' : null,
+            backgroundColor,
+            outlineWidth: 0
+          };
+        }}
+      >
+      <Text>Nested pressables</Text>
+    </Pressable>
+  </Pressable>
+</View>
+```
+
+## Outstanding issues
+There are some outstanding issues that need to be address specifically for desktop scenarios when it comes to fully implementing mouse hover and leave behavior. They are as follows:
+
+- Prevent mouse input from being 'eaten' in ScrollView scenarios [issue #5867](https://github.com/microsoft/react-native-windows/issues/5867)
+- Mouse leave event not firing correctly in Scrollable FlatList scenarios [issue #4614](https://github.com/microsoft/react-native-windows/issues/4614)
+- The hover events/indication for overlapping content prioritizes items "closer" in z-order [issue #5901](https://github.com/microsoft/react-native-windows/issues/5901)
+- Remove the tapped event handler on the `View` component [issue #2349](https://github.com/microsoft/react-native-windows/issues/2349)
+
+## onPointerXX callbacks
+The following callbacks are available on `View` component (and get passed through to Pressable) in Windows and macOS to cover the most common use cases where mouse handling would occur.
+
+The below will need to be implemented for macOS as well [issue TBD]()
+
+| API | Args | Returns | Description |
+|:---:|:----:|:-------:|----|
+| onPointerOver | IPointerEvent | void | Fires when a pointing device is moved within the hit test boundaries of a element. |
```

Comment:
Interested in what all we include on `IPointerEvent`. We currently dump quite a bit into our mouse events that don't necessarily line up with what xplat pointer events should look like.

See https://github.com/microsoft/react-native-windows/blob/master/vnext/src/Libraries/Types/CoreEventTypes.windows.js#L157

```js
// [Windows Mouse events on Windows don't match up with the version in core
// introduced for react-native-web. Replace typings with our values to catch
// anything dependent on react-native-web specific values
export type MouseEvent = SyntheticEvent<
  $ReadOnly<{|
    target: number,
    identifier: number,
    pageX: number,
    pageY: number,
    locationX: number,
    locationY: number,
    timestamp: number,
    pointerType: string,
    force: number,
    isLeftButton: boolean,
    isRightButton: boolean,
    isMiddleButton: boolean,
    isBarrelButtonPressed: boolean,
    isHorizontalScrollWheel: boolean,
    isEraser: boolean,
    shiftKey: boolean,
    ctrlKey: boolean,
    altKey: boolean,
  |}>,
>;
// Windows]
```